developers/blobstream-x-deploy.md

```diff
@@ -6,395 +6,29 @@ next:
-cp -r ../succinctx/plonky2x/verifier/verifier-build .
-# ^ This copies the verifier build to the blobstreamx folder
-mkdir -p build && RUST_LOG=debug cargo run --bin header_range_1024 --release build --wrapper-path verifier-build/ && mv ./target/release/header_range_1024 ./build/header_range_1024
-ls build
-# ^ should return something like: 0x007d0b2a2e2b013612e8.circuit  0x9039e58b2089e5f9abbb.circuit  0xce1636cfaf2bd5497c11.circuit  FunctionVerifier.sol  main.circuit 0x8e1ede4ce0865b41d714.circuit  0xa2140c9bde000dc5e21e.circuit  0xf6759ff933786ddacb92.circuit  header_range_1024
-```
-
-The `header_range_1024` is a specific circuit. Other circuit names can be used there. The current circuits that we have for BlobstreamX: 
-- `header_range_1024`: skip function circuit for batches that are <= 1024 block.
-- `header_range_2048`: skip function circuit for batches that are <= 2048 block.
-- `next_header`: step function circuit.
-
-All the deployments currently rely on two circuits: a header range circuit, either the 1024 or the 2048, depending on the frequency of the batches; The 1024 is mainly for batches that are at a ~1hr frequency and the 2048 for batches that are at a ~3-4hr frequency; and a next header circuit. So, if you're re-building the circuits, make sure to build the correct two circuits for your target deployment.
+2. Follow the deployment instructions in the
+[sp1-blobstream README](https://github.com/succinctlabs/sp1-blobstream?tab=readme-ov-file#deployment).
 
-Now, if you check the `build` folder, you will find a file called `FunctionVerifier.sol`, which is the function verifier contract for your circuit that you can deploy on-chain.
+3. If you're deploying on a chain where there isn't a canonical verifier listed in the
+[SP1 contract addresses](https://github.com/succinctlabs/sp1/blob/main/book/onchain-verification/contract-addresses.md), you'll need to:
 
-Also, you will find the `header_range_1024` under `build` folder which is the binary used to generate the proofs. It is mainly used by the operator to generate the plonky2x proofs that will be PLONK wrapped later using the generated `verifier-build`.
+   a. Deploy your own SP1 Verifier from `sp1-contracts` matching your `sp1-sdk` version.
+   b. Set the `SP1_VERIFIER_ADDRESS` in your `.env` file to the address of your deployed verifier.
 
-At this level, you can deploy the `FunctionVerifier.sol` onchain, then register it in the SuccinctGateway, then use the generated `header_range_1024` circuit and the `verifier` to generate the proofs and submit them onchain. These steps are detailed in the previous section.
+4. To run the prover:
 
-> **_NOTE:_** In a scaleway instance of 64CPU and 504G RAM, the build takes ~10 minutes to complete. Make sure to run it on a beefy machine.
+   - For local proving, set `SP1_PROVER=local` in your environment.
+   - To use the Succinct Proving Network for remote proving, set `SP1_PROVER=network`.
```

Comment:
I'd add the same clarifications here for local proving. E.g. We recommend an instance with 64 vCPU and 128GB of RAM.